e2e/tests/ui_tests/noticeofchange/1v2_lips_to_same_sols_test.js

```diff
@@ -2,6 +2,7 @@ const config = require('../../../config.js');
 const {unAssignAllUsers} = require('../../../api/caseRoleAssignmentHelper');
 const judgeUser = config.judgeUserWithRegionId1;
 
+//Skipping to limit Ft to less than 40 mins - short term skip
```

Comment:
Minor: Is this required?

Reply:
No can be removed. Initially skipped then added it back to see how things are but forgot to remove comments. But will keep it for now. Remove it next pr while enabling stitching api back

Reply:
Otherwise the pr has to rerun again